content/docs/run-api-server/configuring.mdx

```diff
@@ -3,41 +3,59 @@ title: Configuring
+This script invokes Horizon with the `stellar` user, so make sure that permissions for this user are set up accordingly. For example, the `--captive-core-storage-path` (by default the current working directory) should be writable for this user.
+
+</Alert>
+
+Note that the default configuration (located at `/etc/default/stellar-horizon`) provided by the package manager will **enable ingestion by default** (`INGEST=true`). Refer to the later [Ingesting Transactions](./running.mdx#ingesting-transactions) section to see what setup is right for you. If you want certain nodes dedicated exclusively to fulfilling requests, you should drop this flag accordingly.
 
 
 ## Preparing the Database
 
-Before running the Horizon server, you must first prepare the Horizon database. This database will be used for all of the information produced by Horizon, notably historical information about successful transactions that have occurred on the stellar network.
+Before running the Horizon server, you must first prepare the Horizon database specified by the `DATABASE_URL`. This database will be used for all of the information produced by Horizon, most notably historical information about transactions that have occurred on the Stellar network.
 
-To prepare a database for Horizon's use, you must first ensure the database is blank. It's easiest to simply create a new database on your PostgreSQL server specifically for Horizon's use. Next, you must install the schema by running `horizon db init`. Remember to use the appropriate command line flags or environment variables to configure Horizon as explained [above](#parameters). This command will log any errors that occur.
+To prepare a database for Horizon's use, you must first ensure it is blank. It's easiest to create a new database on your PostgreSQL server specifically for Horizon's use (e.g. `createdb horizon`). Note that you may need to [add a role](https://www.postgresql.org/docs/9.6/sql-createrole.html) for yourself through the `postgres` user if you're starting from scratch. Next, install the schema by running `stellar-horizon db init`. This command will log any errors that occur.
+
+Remember to update the appropriate command line flags or environment variables to configure Horizon as explained [above](#parameters).
 
 ### Postgres Configuration
-It is recommended to set `random_page_cost=1` in Postgres configuration if you are using SSD storage. With this setting, Query Planner will make a better use of indices, expecially for `JOIN` queries. We've noticed a huge speed improvement for some queries with this setting.
+It is recommended to set `random_page_cost=1` in Postgres' configuration if you are using SSD storage. With this setting, Query Planner will make a better use of indices, especially for `JOIN` queries. We've noticed a huge speed improvement for some queries with this setting.
 
```

Comment:
from https://github.com/stellar/go/pull/3645
```suggestion
To improve availability of both ingestion and frontend servers it's recommended to set the following values:
* `tcp_keepalives_idle`: 10 seconds,
* `tcp_keepalives_interval`: 1 second,
* `tcp_keepalives_count`: 5.

With the config above, if there are no queries from a given client for 10 seconds, Postgres should start sending TCP keepalive packets. It will retry 5 times every second. If there is no response from the client after that time it will drop the connection.
```